system/service/service.go

```diff
@@ -46,6 +50,17 @@ type (
 		WaitFor(ctx context.Context, ev eventbus.Event) (err error)
 		Dispatch(ctx context.Context, ev eventbus.Event)
 	}
+
+	apiGwService interface {
```

Comment:
We are not using this in unit tests, so why not keep the global service as it was?

Reply:
I hope this will be resolved with the above reply :)